targets/XPlatCppSdk/source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -32,39 +32,52 @@ namespace PlayFab
 
     void PlayFabWinHttpPlugin::WorkerThread()
     {
-        size_t queueSize;
-
-        while (this->threadRunning)
+        try
         {
```

Comment:
Please apply this fix to all other HTTP plugins that have WorkerThreads (Curl, Xbox)